docs/guide/apm-data-model.asciidoc

```diff
@@ -14,56 +14,57 @@ Events can contain additional <<metadata,metadata>> which further enriches your
 [[transaction-spans]]
 === Spans
 
-*Spans* contain information about a specific code path that has been executed.
-They measure from the start to end of an activity,
+*Spans* contain information about the execution of a specific code path.
+They measure from the start to the end of an activity,
 and they can have a parent/child relationship with other spans.
 
-Agents automatically instrument a variety of libraries to capture these spans from within your application.
-In addition, you can use the Agent API for ad hoc instrumentation of specific code paths.
+Agents automatically instrument a variety of libraries to capture these spans from within your application,
+but you can also use the Agent API for custom instrumentation of specific code paths.
 
-A span contains:
+Each span contains:
```

Comment:
There are more fields, but I assume they are not relevant here? (`trace_id`, `subtype`, `action`, `context.db`, ``context.http`, `context.destination`...)

Reply:
I'll add a note that this doesn't include all of the fields in a span. Initially was just trying to give users a feel for what a span could be, without including an exhaustive list.